variables.tf

```diff
@@ -366,6 +366,25 @@ variable "load_balancing_algorithm_type" {
   description = "Determines how the load balancer selects targets when routing requests. Only applicable for Application Load Balancer Target Groups"
 }
 
+variable "load_balancing_anomaly_mitigation" {
```

Comment:
My only thought is it might be more consistent with other modules to use `load_balancing_anomaly_mitigation_enabled` and translate from bool to string inside the module. wdyt?

Reply:
@mschfh apologies - wrong @!

OK, no worries, that's a good point. I guess using bool is mainly beneficial if the values are likely to change, but that seems unlikely in this case.